CIScripts/Test/PesterLogger/RemoteLogCollector.Tests.ps1

```diff
@@ -211,10 +212,32 @@ Describe "RemoteLogCollector - with actual Testbeds" -Tags CI, Systest {
 
     AfterEach {
         Remove-Item "TestDrive:/*" 
-        if (Get-Item function:Write-Log -ErrorAction SilentlyContinue) {
-            Remove-Item function:Write-Log
+        if (Get-Item function:Write-LogImpl -ErrorAction SilentlyContinue) {
+            Remove-Item function:Write-LogImpl
         }
     }
 
     Test-MultipleSourcesAndSessions
+
+    Context "Docker logs" {
+        BeforeEach {
+            Initialize-PesterLogger -OutDir "TestDrive:\"
+        }
+
+        It "captures logs of container" {
+            New-Container -Session $Sess1 -Name foo -Network nat
+
+            Merge-Logs (New-ContainerLogSource -Session $Sess1 -Container foo)
+            $ContentRaw = Get-Content -Raw "TestDrive:\*.Docker logs.captures logs of container.log"
+            $ContentRaw | Should -BeLike "*Microsoft Windows*"
+        }
+
+        It "handles nonexisting container" {
+            Merge-Logs (New-ContainerLogSource -Session $Sess1 -Container bar)
```

Comment:
The problem with:
```powershell
{ Something } | Should -Not -Throw
```
is that in case of an exception being thrown Pester doesn't show the exception and its location at all.

Reply:
Well that's dumb. In this case 
```
{ Something } # | Should -Not -Throw
# Assert is commented out because Pester doesn't show the exception and its location at all
```
would be better